Components/Card/card.css

```diff
@@ -0,0 +1,335 @@
+:root {
```

Comment:
You can create separate `variables.css` file and add all the variables into that file and just import it wherever you want to use them.